SampleApp/SampleApp/Screens/CreatablesView.swift

```diff
@@ -102,15 +123,8 @@ final class CreatableViewModel: ObservableObject {
         campaignAutomation.fetchData(for: location) { [weak self] result in
             switch result {
             case .success(let items):
-                // TODO: CATD-712 : Implement ability to convert CA data to UI
```

Comment:
I think we should keep the old approach and also show the new approach in the Sample App, to show people how to use both.

Reply:
Hi @rossrossp , the old approach is still existed in the function `fetchCreatables()` in this file.
The deletion code here is for refactoring the new function `fetchData()` by using the new interface and `RLTContentConverter`.
Thank you!

Reply:
As agreed - give the sample app a view model which fetches data, then have the view or a presenter convert that data to views

Reply:
This function is inside a viewModel class already for rendering SwiftUI view.
I created another UIKit example and convert data to views in viewModel. 
Please see `CreatablesExampleViewController` and  `CreatablesExampleViewModel`, thank you.